src/main/scala/nl/knaw/dans/easy/agreement/internal/PlaceholderMapper.scala

```diff
@@ -65,19 +65,27 @@ class PlaceholderMapper(metadataTermsFile: File)(implicit parameters: BaseParame
       DansManagedDoi -> doi.getOrElse(""),
       // the following can throw an UnsupportedEncodingException, although this is not expected to ever happen!
       DansManagedEncodedDoi -> doi.map(URLEncoder.encode(_, encoding.displayName())).getOrElse(""),
-      DateSubmitted -> getDate(emd)(_.getEasDateSubmitted).getOrElse(new IsoDate()).toString,
-      Title -> emd.getPreferredTitle
+      DateSubmitted -> dateSubmittedFrom(emd),
+      Title -> escapedTitleFrom(emd)
     )
   }
 
   def sampleHeader(emd: EasyMetadata): Try[PlaceholderMap] = Try {
     Map(
       IsSample -> boolean2Boolean(true),
-      DateSubmitted -> getDate(emd)(_.getEasDateSubmitted).getOrElse(new IsoDate()).toString,
-      Title -> emd.getPreferredTitle
+      DateSubmitted -> dateSubmittedFrom(emd),
+      Title -> escapedTitleFrom(emd)//.replace("&","&amp;")
     )
   }
 
+  private def dateSubmittedFrom(emd: EasyMetadata) = {
+    getDate(emd)(_.getEasDateSubmitted).getOrElse(new IsoDate()).toString
+  }
+
+  private def escapedTitleFrom(emd: EasyMetadata) = {
```

Comment:
Why not a more general `escape` function on strings?

Reply:
You mean on all values of the PlaceholderMap? Even for values placed into href attributes etc?